src/components/Sections/HomepageHeader.tsx

```diff
@@ -15,177 +15,177 @@ import ScarfIcon from "@site/static/img/Homepage/brand/Scarf.svg";
 import IBMIcon from "@site/static/img/Homepage/brand/IBM.svg";
 
 export function HomepageHeader() {
-  return (
-    <div>
-      <img
-        src={HeaderBG}
-        alt=""
-        className="
-      absolute top-[0px] left-[0px] max-h-[580px] 
-      xl:w-screen sm:hidden"
-      />
-      <img
-        src={HeaderBGMB}
-        alt=""
-        className="
-      absolute top-[0px] left-[0px] w-screen z-0
-      sm:h-[496px] mobile:h-[526px]
-      hidden sm:block"
-      />
-      <header
-        className="
-        flex justify-center
-        items-center
-        w-[840px] mobile:w-[90%]  sm:w-[600px]  2xl:w-[1000px]
-        mobile:mt-[32px] sm:mt-[65px] mt-[71px]
-        z-10
-        mx-auto
-        "
-      >
-        <div
-          className="z-10
-          flex flex-col 
-          flex-wrap
-        "
-        >
-          <div
-            className="text-primary
-            font-bold font-inter
-            text-label40 sm:text-heading2 mobile:text-label20
-            sm:font-semibold"
-          >
-            Debug engineering processes and discover opportunities with DORA, Community Growth, Engineering Throughput and more DevLake dashboards...
-          </div>
-          <div
+    return (
+        <div className="w-full">
+          <img
```

Comment:
Hi, @07-Chandra, I understand you only modified `w-full`, but there are still many additional changes in this file, can you remove these? Thank you.